tests/cephfs/cephfs_system/mds_failover_standby_replay_systemic_test.py

```diff
@@ -0,0 +1,366 @@
+        )
+        def_mds_mem = out.strip()
+        crash_status_before = fs_util_v1.get_crash_ls_new(client)
+        log.info(f"Crash status before Test: {crash_status_before}")
+
+        log.info("Set MDS Cache memory limit to 2GB and verify")
+        set_verify_mds_config(client, "2147483648")
+        log.info("Setup test configuration")
+        setup_params = test_setup(fs_util_v1, ceph_cluster, client)
+        sv_list = setup_params["sv_list"]
+        mnt_params = {
+            "client": clients[1],
+            "fs_util": fs_util_v1,
+            "fs_name": setup_params["default_fs"],
+            "export_created": 0,
+        }
+        nfs_params = {
+            "nfs_name": setup_params["nfs_name"],
+            "nfs_export_name": setup_params["nfs_export_name"],
+            "nfs_server": setup_params["nfs_server"],
+        }
+        mnt_info = {}
+        for sv in sv_list:
+            mnt_type = random.choice(["fuse", "kernel", "nfs"])
+            if mnt_type == "nfs":
+                mnt_params.update(nfs_params)
+            cmd = f"ceph fs subvolume getpath cephfs {sv['subvol_name']}"
+            if sv.get("group_name"):
+                cmd += f" {sv['group_name']}"
+            out, rc = client.exec_command(
+                sudo=True,
+                cmd=cmd,
+            )
+            sv_path = out.strip()
+            mnt_client = random.choice(clients)
+            mnt_params.update({"mnt_path": sv_path, "client": mnt_client})
+            path, _ = fs_util_v1.mount_ceph(mnt_type, mnt_params)
+
+            mnt_info.update({sv["subvol_name"]: {"path": path, "client": mnt_client}})
+        out, rc = client.exec_command(
+            cmd=f"ceph fs status {fs_name} -f json", client_exec=True
+        )
+        log.info(out)
+        parsed_data = json.loads(out)
+        for mds in parsed_data.get("mdsmap"):
+            if mds.get("rank") == 0 and mds.get("state") == "active":
+                active_mds = mds["name"]
+        out, rc = client.exec_command(cmd=f"ceph fs status {fs_name}", client_exec=True)
+        out_list = out.split("\n")
+        for out_iter in out_list:
+            if "0-s" in out_iter:
+                standby_replay_mds = out_iter.split()[2]
+        log.info(
+            f"Active MDS : {active_mds}, Standy-by Replay MDS:{standby_replay_mds}"
+        )
+        for mds_nodes_iter in mds_nodes:
+            if mds_nodes_iter.node.hostname in active_mds:
+                active_mds_node = mds_nodes_iter
+        log.info("Start MDS Load to increased MEM and CPU usage by 70%")
+        if (
+            fs_util_v1.mds_mem_cpu_load(
+                client, 70, active_mds, active_mds_node, sv_list, mnt_info
+            )
+            == 1
+        ):
+            log.error("Failed to load MDS to increase MEM and CPU usage by 70%")
+            return 1
+        log.info(
+            "Perform Active MDS failover, and Verify Standby-Replay daemon takesover"
```

Comment:
Can we include all failovers in this along with the mds fail which is already taken care of?
* Reboot
* Daemon stop
* Service stop/start - short duration, long duration
* SIGHUP